connection-service/new-admin-vm/sumo-admin-vm.json

```diff
@@ -1,24 +1,6 @@
 {
 "api.version": "v1",
 "sources": [
-    {
-        "name":  "CAM_Deploy_ConnectionServer_Win_Azure_Logs",
-        "category":  "CAM/Deploy/ConnectionServer/Win/Azure/Logs",
-        "automaticDateParsing":  true,
-        "multilineProcessingEnabled":  true,
-        "useAutolineMatching":  false,
-        "forceTimeZone":  false,
-        "timeZone":  "Etc/UTC",
-        "filters":  [
-                    ],
-        "cutoffTimestamp":  0,
-        "encoding":  "UTF-8",
-        "pathExpression":  "C:\\WindowsAzure\\**\\*.log",

```

Comment:
The DSC logs we use for debugging are somewhere under this path - how do we get those?